controller/backup_controller.go

```diff
@@ -331,6 +331,8 @@ func (bc *BackupController) reconcile(backupName string) (err error) {
 				return err
 			}
 			log.WithError(err).Error("Cannot found the corresponding volume")
+			backup.Status.State = longhorn.BackupStateError
```

Comment:
Suggest updating `Status.Error` as well to better know the cause. You can leverage the above log.

Reply:
Updated, thanks.

Reply:
nit: it's better to reuse the err and customized message instead of just repeating the same message. 

```golang
err = fmt.Errorf("cannot found the corresponding volume: %v", err)
log.WithError(err).Error()
backup.Status.Error = err.Error()
```

or if you want to keep the original err + the customized message when using log
```golang
errMsg := "Cannot found the corresponding volume"
log.WithError(err).Error(errMsg)
backup.Status.Error = errMsg
```

Reply:
Thanks, changed.